### PR TITLE
Upload function 

### PR DESCRIPTION
**Pull request by jnojek**
added upload function and rspec/cucumber testing

### CODE REVIEW

File: app/controllers/uploads_controller.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  def new
+    @upload = Upload.new
+  end
+
+  def create
+    Rails.logger.info(params.inspect)
+    uploaded_file = params[:upload][:csv_file]
+
+      upload_path = Rails.env.test? ? "#{Rails.root}/tmp/test_uploads" : "#{Rails.root}/public/uploads"
+
```
Comment (haridher-tamu) [inline app/controllers/uploads_controller.rb:11]: Might want to run rubocop -a once

File: features/support/simplecov.rb [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 require 'simplecov'
-SimpleCov.start 'rails'
+SimpleCov.start 'rails' do
+  add_filter "/channels/"
```
Comment (haridher-tamu) [inline features/support/simplecov.rb:5]: Thanks for this!!

File: config/herokucredentials.yml.enc
Comment (haridher-tamu) [inline config/herokucredentials.yml.enc:1]: You'll need to rename this back I think. It's kind of annoying to do each time sorry, until we find some other solution to this.

File: app/controllers/uploads_controller.rb [hunk 3]
```diff
@@ -0,0 +1,23 @@
+class UploadsController < ApplicationController
```
Comment (davisbeilue) [inline app/controllers/uploads_controller.rb:1]: Should we have separate controllers for uploads and downloads or combine them into one?

File: features/support/simplecov.rb [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
+  add_filter "/jobs/"
+  add_filter "/mailers/"
+  add_filter "admins_controller"
```
Comment (haridher-tamu) [inline features/support/simplecov.rb:8]: I've added the testcases for this. Infact, just added this file changes as well to run my testcases lol.

**Review by davisbeilue — APPROVED** [review]

**Pull request merged**